Adyen/Core/APIClient/Responses/PaymentStatusResponse.swift

```diff
@@ -1,14 +1,14 @@
 //
-// Copyright (c) 2021 Adyen N.V.
+// Copyright (c) 2022 Adyen N.V.
 //
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
 //
 
 import AdyenNetworking
 import Foundation
 
-/// :nodoc:
 /// Represents the payment status code.
+@_spi(AdyenInternal)
 public enum PaymentResultCode: String, Decodable {
```

Comment:
This one could be API 🤔
Merchant can potentially benefit of it

Reply:
I don't think so, its not our responsibility to provide this API IMO :-)